src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -3174,6 +3184,183 @@ paths:
+        ```json
+        {
+          "entry": {
+            "storageProperties": {
+              "x-alf-archived": "false",
+              "x-amz-storage-class": "INTELLIGENT_TIERING"
+            },
+            "id": "cm:content"
+          }
+        }
+        ```
+        Glacier archive storage class (no restore request ongoing or submitted):
+        ```json
+        {
+          "entry": {
+            "storageProperties": {
+              "x-alf-archived": "true",
+              "x-amz-storage-class": "GLACIER"
+            },
+            "id": "cm:content"
+          }
+        }
+        ```
+        Glacier archive storage class (restore request ongoing, not completed):
+        ```json
+        {
+          "entry": {
+            "storageProperties": {
+              "x-alf-archive-restore-in-progress": "true",
+              "x-amz-restore": "ongoing-request=\"true\"",
+              "x-alf-archived": "true",
+              "x-amz-storage-class": "GLACIER"
+            },
+            "id": "cm:content"
+          }
+        }
+        ```
+        Glacier archive storage class (restore request completed):
+        ```json
+        {
+          "entry": {
+            "storageProperties": {
+              "x-alf-archive-restore-in-progress": "false",
+              "x-amz-restore": "ongoing-request=\"false\", expiry-date=\"Fri Nov 26 01:00:00 CET 2021\"",
+              "x-alf-archive-restore-expiry": "2021-11-26T00:00:00.000Z",
+              "x-alf-archived": "true",
+              "x-amz-storage-class": "GLACIER"
+            },
+            "id": "cm:content"
+          }
+        }
+        ```
+      operationId: getStorageProperties
+      produces:
+        - application/json
+      parameters:
+        - $ref: '#/parameters/nodeIdParam'
+        - $ref: '#/parameters/contentPropNameParam'
+      responses:
+        '200':
+          description: Successful response
+          schema:
+            $ref: '#/definitions/ContentStorageInfo'
+        '400':
+          description: |
+            Invalid parameter: nodeId is not a valid format, or is not a file
+        '401':
+          description: Authentication failed
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+  '/nodes/{nodeId}/storage-info/{contentPropName}/archive':
+    post:
+      x-alfresco-since: "7.2.0"
+      tags:
+        - storage-info
+      summary: Request to send given content to acrhive
+      description: |
+        **Note:** this endpoint is available in Alfresco 7.2.0 and newer versions.
+        It also requires at least one specific implementation of underlying functionality in Cloud Connector(s).
+
+        Request to send given content to acrhive.
+      operationId: requestArchiveContent
+      produces:
+        - application/json
+      parameters:
+        - $ref: '#/parameters/nodeIdParam'
+        - $ref: '#/parameters/contentPropNameParam'
+      responses:
```

Comment:
Should we also get a 403 for users without write permission?

Reply:
Thanks I added 403 and 409 and some more description to 400 as well.